src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -191,7 +191,8 @@
                 .filter(Objects::nonNull).collect(Collectors.toList());
             processedCwProfiles = persistCaseWorkerInBatch(newCaseWorkerProfiles, updateCaseWorkerProfiles, requestMap);
         } catch (Exception exp) {
-            log.error("{}:: createCaseWorkerUserProfiles failed ::{}", loggingComponentName, exp);
+            log.error("{}:: createCaseWorkerUserProfiles failed :: Job Id {} ::{}", loggingComponentName,
+                    validationServiceFacade.getAuditJobId(),exp);
```

Comment:
Not sure whether checkstyle is kicking in. Ideally, there should be a space between "," and "exp"

Reply:
Ideally there should be space. Ran checkstyle locally, did not find any issues. I could see only below rule in checkstyle
`<module name="NoWhitespaceBefore">
      <property name="tokens"
                value="COMMA, SEMI, POST_INC, POST_DEC, DOT, ELLIPSIS, METHOD_REF"/>
      <property name="allowLineBreaks" value="true"/>
    </module>`